packages/fxa-content-server/app/scripts/views/settings.js

```diff
@@ -109,6 +117,8 @@ const View = BaseView.extend({
     const account = this.getSignedInAccount();
 
     context.set({
+      ccExpired: !!this._ccExpired,
+      escapedCcExpiredLinkAttrs: `href="${this._subscriptionManagementUrl}" class="alert-link"`,
```

Comment:
This payments-server URL doesn't currently work as a plain link - it needs an access token to enable auth-server API access.

We just landed PR #1899 for issue #1850 to support linking to subscription management. The link URL here would need to be `{content-server base URL}/subscriptions/`. That will generate an access token and then redirect to subscription management on payments-server.

Reply:
Great, thanks. This is what I was asking about in https://github.com/mozilla/fxa/pull/1846#discussion_r305427026, I had a hunch what I'd done might be wrong.